notebooks/404-style-transfer-webcam/404-style-transfer.ipynb

```diff
@@ -0,0 +1,457 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Style Transfer with OpenVINO™\n",
+    "\n",
+    "This notebook demonstrates style transfer with OpenVINO, using the Style Transfer Models from [ONNX Model Repository](https://github.com/onnx/models), specifically [Fast Neural Style Transfer](https://github.com/onnx/models/tree/master/vision/style_transfer/fast_neural_style) model is one of the style transfer models designed to mix the content of an image with the style of another image. \n",
+    "\n",
+    "This notebook uses five pre-trained models, for the following styles: Mosaic, Rain Princess, Candy, Udnie and Pointilism. The models are from the [ONNX Model Repository](https://github.com/onnx/models) and are based on the research paper [Perceptual Losses for Real-Time Style Transfer and Super-Resolution](https://arxiv.org/abs/1603.08155) along with [Instance Normalization](https://arxiv.org/abs/1607.08022). Final part of this notebook shows live inference results from a webcam. Additionally, you can also upload a video file.\n",
```

Comment:
I suggest including an image sample per model. What the user is expecting to see? One image per style

Reply:
This is a good idea! Anisha, you could use your own image in different styles

Reply:
As recommended I have placed the photo with all style results.

> @anisha -- Please update all changes I provided today.

@raymondlo84  Request to please review the changes I have made based on your review comments - Reference formatting, preprocessing step description.

Reply:
That is a great add, thanks Anisha. It looks amazing.